pages/common/nano.md

```diff
@@ -1,11 +1,15 @@
 # nano
 
-> Simple, easy to use editor. An enhanced, free Pico clone.
+> Simple, easy to use command-line text editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
-- Start nano in terminal with {filename}:
+- Open a specific file:
```

Comment:
```suggestion
- Open a file:
```

Reply:
I don't think this is an improvement, it's correct to say that the file is the one specified, and not just "a file".

Reply:
Okay, I edited it.

Reply:
Still no improvement though. "Open a specific file" sounds better to me than "Open the specified file". It describes the general operation, using "a" instead of "the". We use "a specific" a lot more than "the specified" through all pages.

Reply:
Perhaps you're right. I judged this according to some recent edits, so my view is probably skewed.